integrations/launchdarkly/.port/resources/blueprints.json

```diff
@@ -153,5 +153,32 @@
         "many": false
       }
     }
+  },
+  {
+    "identifier": "featureFlagInEnvironment",
+    "description": "This blueprint represents a feature flag in LaunchDarkly Environment.",
+    "title": "Feature Flag In Environment",
+    "icon": "Launchdarkly",
+    "schema": {
+      "properties": {},
+      "required": []
+    },
+    "mirrorProperties": {},
```

Comment:
Let's add here relevant mirror properties specifically from the `launchDarklyFeatureFlag` blueprint, such as the kind, description and deprecated fields